View/encoding_table.py

```diff
@@ -0,0 +1,36 @@
+from PySide6 import QtWidgets
+from PySide6.QtWidgets import QTableWidget
+
+
+class EncodingTable(QTableWidget):
+    """
+    EncodingTable is a custom QTableWidget used to support encoding table
+    functionalities in the front-end.
+    """
+
+    def __init__(self):
+        """
+        Constructor - Sets the properties of a QTableWidget.
+        """
+        super().__init__()
+
+        self.setRowCount(10)
+        self.setColumnCount(4)
+
+        self.horizontalHeader().setStretchLastSection(True)
+        self.horizontalHeader().setSectionResizeMode(QtWidgets.QHeaderView.Stretch)
+        self.verticalHeader().setStretchLastSection(True)
+        self.verticalHeader().setSectionResizeMode(QtWidgets.QHeaderView.Stretch)
```

Comment:
Could add a comment here about what you are doing

Reply:
These changes show as "added" since I renamed the file. The code you're referencing is from Jacob's commit from the last sprint. @Jcobo314 if you have any ideas for a comment here, I'll put them here. Otherwise, it can be changed in another patch.